Frameworks/Foundation/NSCache.mm

```diff
@@ -161,7 +161,7 @@ - (void)removeObjectForKey:(id)key {
 */
 - (void)removeAllObjects {
     @synchronized(self) {
-        for (auto& entry: _cacheEntries) {
+        for (auto& entry : _cacheEntries) {
             if (_delegateFlags.hasWillEvict) {
```

Comment:
looks like clang formatting

---
In reply to: [70347357](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70347357) [](ancestors = 70347357)